applications/crossbar/priv/couchdb/schemas/system_config.navi.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "system_config.navi",
+    "description": "Schema for navi system_config",
+    "properties": {
+        "notification_servers": {
+            "description": "all apps that users can subscribe to push notifications from",
+            "items": {
+                "anyOf": [
+                    {
+                        "properties": {
+                            "notification_type": {
+                                "enum": [
+                                    "apns"
+                                ]
+                            }
+                        },
+                        "required": [
+                            "certificate",
```

Comment:
Seems strange to require keys but not define their properties? What's happening here?

Reply:
The keys had descriptions further along in the schema, they are only simple strings. To improve the readability of the schema I have re-templated it so that the descriptions for the required keys for either branch of the anyOf are in that subschema.